github/multiformats.yml

```diff
@@ -2548,6 +2551,14 @@ teams:
       member:
         - lukehoersten
     privacy: closed
+  Helia Dev:
+    # This should match https://github.com/orgs/ipfs/teams/helia-dev
```

Comment:
We can codify this! We can add a rule to https://github.com/multiformats/github-mgmt/blob/master/scripts/src/actions/fix-yaml-config.ts that would look up the team members at https://github.com/orgs/ipfs/teams/helia-dev and set them here too.

Reply:
Do we still want to do this?